WordPress/Classes/Networking/SiteServiceRemoteREST.h

```diff
@@ -3,9 +3,15 @@
 NS_ASSUME_NONNULL_BEGIN
 
 @interface SiteServiceRemoteREST : ServiceRemoteREST
+
 @property (nonatomic, readonly) NSNumber *siteID;
+
 - (instancetype)initWithApi:(WordPressComApi *)api __unavailable;
 - (instancetype)initWithApi:(WordPressComApi *)api siteID:(NSNumber *)siteID;
+
+- (instancetype)initWithComRestApi:(WordPressComRestApi *)api __unavailable;
```

Comment:
I copied this from the api method, I think it's to avoid the use of this init method that is defined on the base class.